lib/synapse_pay_rest/client.rb

```diff
@@ -30,7 +30,8 @@ class Client
     # @param log_to [String] (optional) file path to log to file (logging must be true)
     def initialize(client_id:, client_secret:, ip_address:, fingerprint: nil,
                    development_mode: true, **options)
```

Comment:
tiny, why don't we just put in `base_url: nil, **options` instead of the line below?

Reply:
Fixed 👍